### PR TITLE
Showing exchange rate for cryptos

### PR DESCRIPTION
**Pull request by rockstardev**
Ref: https://github.com/btcpayserver/btcpayserver/pull/170#issuecomment-389462155

### CODE REVIEW

**Comment by rockstardev** [comment]
Here is how it looks with changes:

showing exchange rate

File: BTCPayServer/Controllers/InvoiceController.UI.cs [hunk 1]
```diff
@@ -314,18 +323,13 @@ public static string FormatCurrency(decimal price, string currency, CurrencyName
...
-                return null;
-            }
 
-            return FormatCurrency(productInformation.Price, productInformation.Currency, _CurrencyNameTable);
+            if (_specialCryptoFormat.Contains(currency))
+                return price.ToString("C", provider);
+            else
+                return price.ToString("C", provider) + $" ({currency})";
         }
+        private static readonly string[] _specialCryptoFormat = new[] { "BTC", "LTC" };
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/InvoiceController.UI.cs]: wat? why only BTC and LTC?

Comment (NicolasDorier) [reply BTCPayServer/Controllers/InvoiceController.UI.cs]: Fixed https://github.com/btcpayserver/btcpayserver/commit/eeb522fe7d852e610e1c4b0ded7fc56ffa229ce1

Comment (rockstardev) [reply BTCPayServer/Controllers/InvoiceController.UI.cs]: Yeah, much better that way - was going to ask you if you were already maintaining that data somewhere.

**Pull request merged**